forward/forwarder_test.go

```diff
@@ -145,8 +255,7 @@ func (s *ForwarderTestSuite) TestKeysDiverged() {
 func (s *ForwarderTestSuite) TestRequestTimesOut() {
 	var ping Ping
 
-	s.sender.lookup = "127.0.0.2:3001"
-	dest, err := s.sender.Lookup("some key")
+	dest, err := s.sender.Lookup("unreachable")
```

Comment:
Could this get funky if someone *is* listening on that port?  Should we listen, connect, and fail to reply?